ui/main.qml

```diff
@@ -159,18 +159,8 @@ StatusWindow {
     Connections {
         target: applicationWindow
         onClosing: {
-            if (loader.sourceComponent == login) {
-                applicationWindow.visible = false
-                close.accepted = false
-            }
-            else if (loader.sourceComponent == app) {
-                if (appSettings.quitOnClose) {
-                    close.accepted = true
-                } else {
-                    applicationWindow.visible = false
-                    close.accepted = false
-                }
-            }
+            loader.sourceComponent = undefined
```

Comment:
This may cause QML warning probably. Why not `loader.active = false`?

Reply:
Will see, we can easily change that if there is a warning, I didn't face it so far. The reason why I did that is that I wanted to unload the component (you can check by setting the line bellow to `close.accepted = false` and closing the app that component is unloaded properly), setting `loader.active` to `false` has no point because we're quitting the app in that moment.